src/java/io/bazel/rulesscala/scalac/BUILD

```diff
@@ -6,6 +6,12 @@ java_binary(
         "ScalacProcessor.java",
     ],
     main_class = "io.bazel.rulesscala.scalac.ScalaCInvoker",
+    # this probably should be set globally for the entire rules_scala
```

Comment:
Maybe this needs to be in the scala.bzl?
If so we need to also test that the user gets a clear error message when they request a lower version. 
And also they can request a higher version and it's ok

Reply:
Is there a pattern of enforcing minimal Java version on clients of the rule?